launcher: improve .rom file handling

macmpi opened the pull request:
- better capture and handle missing -r option (also in cmdline)
- give clue on expected path if missing rom
- return errors as such (exit code and messages in &2)
- allow multi-lines messages for zenity

mike632t commented on src/x11-calc.sh.in:
```diff
@@ -119,7 +120,8 @@ _launch() {
          ;;
       $ENOENT)
-         _errmsg="Missing ROM file!"
+         _errmsg="`echo "$OPTIONS" | awk -F '-r ' '{ print $2 }' | awk -F '-. ' '{ print $1 }'`"
```
A multi line error message works fine in a message box but not so well on the command line, particularly if the output is being redirected to a log file (which I admit is unlikely in this case).  Perhaps define _errmsg for the short message, and _errhelp with a longer explanation to use if zenity is available ?

macmpi replied:
Not sure about the best way to handle, as user on cmdline would aslo appreciate having a more detailed clue on how to fix the error.
Thinking of which... launcher should probably send error messages to `>&2` and return error code on exit.

macmpi replied:
Tried some (interim) trick: thoughts?

macmpi replied:
added double-space to new-line conversion to allow expanded messages both in cmdline and zenity.

macmpi committed:
launcher: improve .rom file handling

- better capture and handle missing -r option (also in cmdline)
- give clue on expected path if missing rom
- return errors as such (exit code and messages in &2)
- allow multi-lines messages for zenity
